core/computers.go

```diff
@@ -105,6 +109,30 @@ func GetPercentageOfValue(value *big.Int, percentage float64) *big.Int {
 	return result
 }
 
+// GetIntPercentageOfValue returns the exact percentage of value, that fits into the integer (with loss of division remainder)
+func GetIntTrimmedPercentageOfValue(value *big.Int, percentage float64) *big.Int {
+	x := big.NewInt(0).Set(value)
+	percentageString := strconv.FormatFloat(percentage, 'f', -1, 64)
+	exp, fra := splitExponentFraction(percentageString)
+	concatExpFra := append([]rune(exp), []rune(fra)...)
```

Comment:
why casting to rune? `exp` and `fra` should contain only numbers (ASCII characters between 0x30-0x39). I'm sure it will work without casting to rune and the cast on line 118 will become useless.
```GO
concatExpFra := exp + fra
```